playground_project/nodestream.yaml

```
@@ -0,0 +1,27 @@
+scopes:
+  test:
+    pipelines:
+      - test.yaml
+      - test2.yaml
+
+targets:
+  db-one:
+    database: neo4j
+    database_name: neo4j
+    username: neo4j
+    password: neo4j123
+    uri: bolt://localhost:7687
+  db-two:
+    database: neo4j
+    database_name: neo4j
+    username: neo4j
+    password: neo4j123
+    uri: bolt://localhost:7688
+  db-nept:
```

```suggestion
  db-neptune:
```

nit: spell full name - neptune